components/tools/OmeroPy/test/integration/test_permissions.py

```diff
@@ -975,7 +887,7 @@ def assertPerms(self, perms, fixture):
         assert expected_arr == found_arr
 
     @pytest.mark.broken(ticket="12474")
-    @pytest.mark.parametrize("fixture", PFS)
+    @pytest.mark.parametrize("fixture", lib.PFS)
```

Comment:
No I should likely use `ids=` in this parametrization here as well if that makes sense.